README.md

```diff
@@ -1,2 +1,185 @@
-# irma-fab-integration
-This repository contains a platform to integration IRMA and FAB
+![Checks](https://github.com/InternetNZ/irma-fab-integration/actions/workflows/checks.yml/badge.svg)
+
+# IRMA FAB Integration
+This repository contains FAB relying-party APIs also a CLI tool in order to work with FAB and IRMA.
+
+## FAB Relying-Party API
+The relying-party’s API is what the FAB will call when passing the credentials the end user has requested to share.
+The credentials will be shared in the W3C verifiable credential format. This specification can be found
+[here](https://www.w3.org/TR/vc-data-model/).
+
+### How to run it on your local
+First, make sure to create `.env` file from `.env.sample` and set the value for `SINGLE_SOURCE_API_KEY` environment
```

Comment:
Maybe a typo? Single source is not the case, right?

Reply:
Not solved yet.

Reply:
done